packages/vuetify/src/components/VTreeview/VTreeviewItem.sass

```diff
@@ -14,17 +14,75 @@
         width: 0
 
   .v-treeview
+    --v-indent-line-width: #{$treeview-indent-line-width}
+    --v-indent-line-width-half: round(var(--v-indent-line-width) / 2, 1px)
+    --v-tree-indent: var(--indent-padding)
     &.v-list
       --indent-padding: #{$treeview-indent-padding}
 
     &.v-list--disabled .v-list-item__prepend
       pointer-events: auto
 
+    .v-list-item--slim > .v-list-item__prepend
+      > .v-icon ~ .v-list-item__spacer
+        width: $treeview-item-spacer-width
```

Comment:
A bit out of scope, but I think it makes a big difference and should land in 3.9.0 separately if this PR is not going to make it.